src/sources/util/unix.rs

```diff
@@ -58,6 +63,7 @@ pub fn build_unix_source(
                 let received_from: Option<Bytes> =
                     path.map(|p| p.to_string_lossy().into_owned().into());
                 let lines_in = FramedRead::new(socket, LinesCodec::new_with_max_length(max_length))
+                    .take_until(shutdown.clone())
```

Comment:
Seems this is an extra clone? The one above is necessary, but I don't see it used later.

Reply:
It is needed, since we are accepting multiple connections/sockets here.